### PR TITLE
Create blog-guidelines.md

### PR DESCRIPTION
**Pull request by xmulligan**
Signed-off-by: Bill Mulligan <billmulligan516@gmail.com>

### CODE REVIEW

File: blog-guidelines.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+* The most interesting posts are how-to blogs, blogs with technical content, and those that teach or show how to do something in a way others may not have thought of.
+* Blog posts that show hurdles that were encountered and explain how they were overcome often do very well as the community is looking to learn.
+* Content on how to choose between different technologies and how to accommodate different legacy issues and platforms is a good way to highlight your learnings and share with the community.
+* When showing upstreaming of a patch fixing an issue for others, link back to the Github issue, so readers can follow along.
+* Critical commentary or broad issues must be approached with sensitivity, professionalism, and tact in a way that is beneficial and positive for the community.
+* Your post must be your content. We are also happy to link to external blog posts if they are published elsewhere.
```
Comment (xmulligan) [inline blog-guidelines.md:28]:
```suggestion
* Your post must be your content. Everything published on ebpf.io will be under the [Creative Commons Attribution 4.0 International License](https://creativecommons.org/licenses/by/4.0/). Your post can be republished to another website under those terms.
* We are also happy to link to external blog posts if they are published elsewhere.
* Translations of blog post are also accepted.
```

File: blog-guidelines.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+Authors can add 1-2 sentences at the end of their blog with a link to an external site such as their website or a gated asset. Examples for best practices:
+* Share contact info of  speaker - twitter, slack, email
+* Links to company websites
+* Links to projects websites or Github
+* Can advertise if they are hiring and who to contact if interested
+
+We will review the statement, as we do for the content for the blog, to approve the content. If changes are needed, we will provide suggested changes. 
+
+## Promotion ##
+Please feel free to share the blog on Twitter and reach out to [blog@ebpf.io](blog@ebpf.io) if you would like to request ebpf.io's handle RTs.
```
[the remark this answers is not in the document]
Comment (xmulligan) [reply blog-guidelines.md:41]: @ebpfsummit

[the remark this answers is not in the document]
Comment (qmonnet) [reply blog-guidelines.md:41]: OK, the phrasing was not clear to me. Maybe: “... if you would like to request retweets from @ebpfsummit” or “from the account associated to ebpf.io”.

File: blog-guidelines.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+Authors can add 1-2 sentences at the end of their blog with a link to an external site such as their website or a gated asset. Examples for best practices:
+* Sharing contact info of the author: Twitter, Slack, email...
+* Links to company websites
+* Links to projects websites or GitHub
+* Advertising if they are hiring and who to contact if interested
+
+We will review the statement, as we do for the content for the blog. If changes are needed, we will provide suggestions.
+
+## Promotion ##
+Please feel free to share the blog on Twitter and reach out to [blog@ebpf.io](blog@ebpf.io) if you would like to request ebpf.io's handle RTs.
```
Comment (xmulligan) [inline blog-guidelines.md:41]:
```suggestion
Please feel free to share the blog on Twitter and reach out to [blog@ebpf.io](blog@ebpf.io) if you would like to request retweets from [@ebpfsummit](https://twitter.com/eBPFsummit).
```

File: blog-guidelines.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+* Blog posts that show hurdles that were encountered and explain how they were overcome often do very well as the community is looking to learn.
+* Content on how to choose between different technologies and how to accommodate different legacy issues and platforms is a good way to highlight your learnings and share with the community.
+* When showing upstreaming of a patch fixing an issue for others, link back to the GitHub issue (or upstream patch, mailing list discussion, etc.), so readers can follow along.
+* Critical commentary or broad issues must be approached with sensitivity, professionalism, and tact in a way that is beneficial and positive for the community.
+* Your post must be your content. Everything published on ebpf.io will be under the [Creative Commons Attribution 4.0 International License](https://creativecommons.org/licenses/by/4.0/). Your post can be republished to another website under those terms.
+* We are also happy to link to external blog posts if they are published elsewhere.
+* Translations of blog post are also accepted.
```
Comment (qmonnet) [inline blog-guidelines.md:30]: “a blog post” or “blog posts”?

**Review by qmonnet — APPROVED** [review]

File: blog-guidelines.md (hunk 4 above)
Comment (xmulligan) [inline blog-guidelines.md:30]:
```suggestion
* Translations of blog posts are also accepted.
```

File: blog-guidelines.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
+# ebpf.io Blog Guidelines #
+
+## ebpf.io Blog Overview ##
+
+The ebpf.io (CNCF) blog serves as a channel for eBPF project maintainers, contributors, and anyone interested in eBPF to share content with the eBPF community.
```
Comment (borkmann) [inline blog-guidelines.md:5]: Why the `(CNCF)`? Might be bit confusing since it's not a CNCF project.

Comment (borkmann) [reply blog-guidelines.md:5]: Otherwise lgtm.

Comment (xmulligan) [inline blog-guidelines.md:5]:
```suggestion
The ebpf.io blog serves as a channel for eBPF project maintainers, contributors, and anyone interested in eBPF to share content with the eBPF community.
```

Comment (xmulligan) [reply blog-guidelines.md]: Seems I missed that part in the copy paste. I've removed now

File: blog-guidelines.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+* Technical content and how-to’s
+* Stories about eBPF project deployments
+* Use cases and success stories
+* Industry insight into eBPF adoption
+* Reports from eBPF related events
+
+Other topics are welcome, but it needs to have a direct link to the eBPF community. We accept content from anyone and external links are welcome.
+
+## Content ##
+The CNCF blog audience is developers, IT operators, and eBPF and open source enthusiasts. Keep this in mind as you develop your blog content.
```
Comment (borkmann) [inline blog-guidelines.md:18]: nit: CNCF -> ebpf.io

**Pull request merged**